addon/components/google-maps-markup/template.hbs

```diff
@@ -38,6 +38,32 @@
           {{color-palette selected=(get activeTool opt.id) onselect=(action 'updateOptionValue' activeTool opt.id)}}
         </div>
       {{/if}}
+      {{#if (is-equal opt.type 'icon')}}
+        <div class="dropdown {{if iconDropDown 'open'}}">
+          <button class="btn btn-default dropdown-toggle" type="button" id="iconDropDown" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true" {{action 'toggleDropdown' 'iconDropDown'}}>
+            {{activeTool.icon.display}}
+            <span class="caret"></span>
+          </button>
+          <ul class="dropdown-menu" aria-labelledby="iconDropDown" {{action 'toggleDropdown' 'iconDropDown'}}>
+            {{#each activeTool.icons as |icon|}}
+              <li {{action 'updateOptionValue' activeTool 'icon' icon}}><a>{{icon.display}}</a></li>
+            {{/each}}
+          </ul>
+        </div>
+      {{/if}}
+      {{#if (is-equal opt.type 'customColor')}}
+        <div class="dropdown {{if customColorMenu 'open'}}">
```

Comment:
consistent naming between `iconDropDown` and `customColorMenu`, since both are dropdowns